<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SweetCalculator</class>
 <widget class="QDialog" name="SweetCalculator">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sweet Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 170, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="3">
    <widget class="QPushButton" name="minbtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(127, 3, 36);
background-color: rgb(200, 0, 80);
font: 14pt &quot;System&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
border-radius: 12px
</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="num6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 85, 127);
font: 14pt &quot;System&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
border-radius: 12px
</string>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="num2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 85, 127);
font: 14pt &quot;System&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
border-radius: 12px
</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="num4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 85, 127);
font: 14pt &quot;System&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
border-radius: 12px
</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="num9">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 85, 127);
font: 14pt &quot;System&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
border-radius: 12px
</string>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="dotbtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 85, 127);
font: 14pt &quot;System&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
border-radius: 12px
</string>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="revbtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 85, 127);
font: 14pt &quot;System&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
border-radius: 12px
</string>
     </property>
     <property name="text">
      <string>+/-</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="num3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 85, 127);
font: 14pt &quot;System&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
border-radius: 12px
</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="num7">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 85, 127);
font: 14pt &quot;System&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
border-radius: 12px
</string>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="clearbtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 85, 127);
background-color: rgb(255, 29, 59);
font: 14pt &quot;System&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
border-radius: 12px
</string>
     </property>
     <property name="text">
      <string>C</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="mulbtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(127, 3, 36);
background-color: rgb(200, 0, 80);
font: 14pt &quot;System&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
border-radius: 12px
</string>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="num5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 85, 127);
font: 14pt &quot;System&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
border-radius: 12px
</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="sumbtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(127, 3, 36);
background-color: rgb(200, 0, 80);
font: 14pt &quot;System&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
border-radius: 12px
</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="num1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 85, 127);
font: 14pt &quot;System&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
border-radius: 12px
</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="autoRepeatInterval">
      <number>107</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="num0">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 85, 127);
font: 14pt &quot;System&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
border-radius: 12px
</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="divbtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(127, 3, 36);
background-color: rgb(200, 0, 80);
font: 14pt &quot;System&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
border-radius: 12px
</string>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="num8">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 85, 127);
font: 14pt &quot;System&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
border-radius: 12px
</string>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="numtan">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 52, 89);
font: 14pt &quot;System&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
border-radius: 12px
</string>
     </property>
     <property name="text">
      <string>tan</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="numcos">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 52, 89);
font: 14pt &quot;System&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
border-radius: 12px
</string>
     </property>
     <property name="text">
      <string>cos</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="numcot">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 52, 89);
font: 14pt &quot;System&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
border-radius: 12px
</string>
     </property>
     <property name="text">
      <string>cot</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="numsin">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 52, 89);
font: 14pt &quot;System&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
border-radius: 12px
</string>
     </property>
     <property name="text">
      <string>sin</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QPushButton" name="numrad">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 52, 89);
font: 14pt &quot;System&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
border-radius: 12px
</string>
     </property>
     <property name="text">
      <string>Radical</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="numfac">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 52, 89);
font: 14pt &quot;System&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
border-radius: 12px
</string>
     </property>
     <property name="text">
      <string>factorial</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QPushButton" name="eqbtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 85, 127);
font: 14pt &quot;System&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
border-radius: 12px
</string>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QLineEdit" name="textbox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 134, 166);
border-radius: 12px
</string>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::VisualMoveStyle</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
